src/styles/blocks/contacts.scss

```diff
@@ -0,0 +1,132 @@
+.contacts {
+
+  &__title {
+    margin-bottom: 48px;
+  }
+
+  &__wrapper {
+    @include pageGrid;
+    row-gap: 48px;
+  }
+
+  &__wrap {
+    grid-column: span 2;
+
+    @include onDesktop {
+      grid-column: span 6;
+      display: flex;
+      justify-content: space-between;
+    }
```

Comment:
add mixin onTablet with grid-column: span 3. Because as you can see according to your figma design the block has 3 columns not 2 on tablet